### PR TITLE
Create cn-usci.json closes #363

### LINKED ISSUE #363: PROPOSAL: new list PSB for China
## Proposal

**Public Security Bureau**

**PSB**

**Quote received from our colleagues at Plan International China: "The Overseas NGO Management Law became effective in China since January 2017. Main step involved for getting an organization registered under the new law is to identify an appropriate Professional Supervision Unit (PSU) whose area of work is relevant to INGOs’ key area of interventions and the PSU’s willingness to take on the responsibility for becoming that INGOs’ supervisory agency (PSU). Once such an approval is obtained, then INGOs can get their registration done with the Public Security Bureau (PSB)." I have understood that every NGO has to have their registration checked and reconfirmed every year**

**China**

**INGOs**

**Any specific sectors that the list covers**

**A URL for information on the list, and a URL for looking up identifiers (if available)**

**Any information on available open data copies of this list**

**Plan International China uses this ID: "统一社会信用代码：G1610000713522976L". The Chinese part translates to Unified Social Credit Code. Another number that appears often is 00004203 and it's not entirely clear to me which one is the best one to use, and which one refers to the PSB. I'm attaching two documents as examples, and the english text of the law from 2017. I will ask for clarification about these IDs as soon as the China office opens again.
[Overseas NGO Management Law.pdf](https://github.com/org-id/register/files/3686116/Overseas.NGO.Management.Law.pdf)
Plan CHN registration
Plan CHN annual check-2019
**

### PR DESCRIPTION
**Pull request by kd-ods**
This list was researched by @theaschepers.

China's USCIs are universally applied to both domestic and non-domestic actors who are economically or socially active in China. They can be minted by several agencies, though. And there doesn't appear to be a single agency that has a searchable database of USCIs across sectors; hence the various platforms and URLs presented in the list details.

### CODE REVIEW

File: lists/cn/cn-usci.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "publicDatabase": "http://www.chinanpo.gov.cn/search/orgindex.html",
+    "guidanceOnLocatingIds": "This list only shows non-domestic NGOs. Your browser may be able to translate the page if necessary. First field in the form is the name of the organisation. Click the blue button and a new table with results should appear, with the Unified Social Credit Identifier in the third column.\nFor other domestic entities, ask the organisation for their USCI or go to http://gsxt.gov.cn. You will need the name of the organisation in Chinese.",
+    "exampleIdentifiers": "G1610000713522976L, G1330000MCW06863XQ, G11100007178352756",
+    "languages": ["zh"]
+  },
+  "data": {
+    "availability": [],
+    "dataAccessDetails": "",
+    "features": [],
+    "licenseStatus": "open_license",
```
Comment (siwhitehouse) [inline lists/cn/cn-usci.json:35]: I can't see where an open license is granted. An [example page](https://ngo.mps.gov.cn/ngo/portal/viewInfogsNew.do?p_creditcode=G1610000713522976L&p_createdate=2017-06-30&p_tab=B&p_topmenu=3) includes a Copyright statement and the [Site Statement](https://ngo.mps.gov.cn/ngo/portal/viewJsp.do?p_jsp=2&p_topmenu=1) states that 

> The content of this website is provided by the Ministry of Public Security and the Overseas NGO Management Office of Provincial Public Security Organs. Any media, Internet sites and commercial organizations are not allowed to use the content published on this website for commercial original reprinting, nor can they distort or tamper with the content published on this website. The copyright involved in this website belongs to this website.
...
Any media or Internet site shall not reprint the information and service content provided by other units on this website without authorization. If reprinting is required, it must directly contact the corresponding provider to obtain legal authorization.

File: lists/cn/cn-usci.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+   "availableOnline": true,
+    "onlineAccessDetails": null,
+    "publicDatabase": "http://www.chinanpo.gov.cn/search/orgindex.html",
+    "guidanceOnLocatingIds": "This list only shows non-domestic NGOs. Your browser may be able to translate the page if necessary. First field in the form is the name of the organisation. Click the blue button and a new table with results should appear, with the Unified Social Credit Identifier in the third column.\nFor other domestic entities, ask the organisation for their USCI or go to http://gsxt.gov.cn. You will need the name of the organisation in Chinese.",
+    "exampleIdentifiers": "G1610000713522976L, G1330000MCW06863XQ, G11100007178352756",
+    "languages": ["zh"]
+  },
+  "data": {
+    "availability": [],
+    "dataAccessDetails": "",
```
Comment (siwhitehouse) [inline lists/cn/cn-usci.json:33]: I think we could possibly include the following, but you might want to check my reasoning first.
    "features": [
      "registered_address",
      "industry_classifications",
      "legal_representatives",
      "registration_number",
      "tax_id_number",
      "objectives_purpose",
      "expiry_date"
    ],

File: lists/cn/cn-usci.json (hunk 1 above)
Comment (kd-ods) [reply lists/cn/cn-usci.json:35]: Will change to "closed_license".

File: lists/cn/cn-usci.json (hunk 2 above)
Comment (kd-ods) [reply lists/cn/cn-usci.json:33]: Good call. I'm not entirely sure what the date on the records is, so will skip the 'expiry_date' value.

**Pull request merged**